runtime/jcl/common/java_lang_StackWalker.cpp

```diff
@@ -218,6 +218,12 @@ Java_java_lang_StackWalker_getImpl(JNIEnv *env, jobject clazz, jlong walkStateP)
 			}
 			J9VMJAVALANGSTACKWALKERSTACKFRAMEIMPL_SET_METHODNAME(vmThread, PEEK_OBJECT_IN_SPECIAL_FRAME(vmThread, 0), nameString);
 
+			nameString = utfToStringObject(env, J9ROMMETHOD_GET_SIGNATURE(romClass, romMethod), J9_STR_INTERN);
```

Comment:
Why does this assign a signature to `nameString`?  A new local would be clearer

Reply:
Done.
I use "nameString" as a general purpose buffer for several strings (most of them are names of something).  Ideally I should change the variable name to something more generic like "stringObject", but I'll skip that to keep the present changeset small.